code/index.html

```diff
@@ -3,28 +3,162 @@
+              Det viktiga är din inställning och öppenhet eftersom det här är ett nätverk för kvinnor som vet att 1 + 1 = 3; att vi tillsammans blir starkare! 
+            </p>
+          </div>
       </div>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <div class="accordion-item">
+        <div class="accordion-item-header">
+          <h4> Hur ofta ses vi? </h4>
+        </div>
+          <div class="accordion-item-body">
+            <p class="accordion-item-body-content">
+              Vi kommer att träffas minst en gång i månaden. Höstens planering kommer dyka upp här på webbplatsen inom kort. 
+              Stay tuned för mer info! 
+            </p>
+          </div>
       </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
+
+      <div class="accordion-item">
+        <div class="accordion-item-header">
+          <h4>Vad kostar ett medlemsskap?</h4>
+        </div>
+          <div class="accordion-item-body">
+            <p class="accordion-item-body-content">
+              Ett årsmedlemsskap i EDGE kostar 500 kr, prova på i en månad för 150 kr.
+            </p>
+          </div>      
+        </div>
+    </div>
+    <script src="./script.js"></script> <!-- Link for JavaScript file -->
+    <br>
+    <br>
+
+    <!-- Signup form -->
+    <h2>KONTAKT</h2>
+    
+    <div class="container">
+      <form action="http://httpbin.org/anything" method="POST">
+        <div class="row">
+          <div class="col-25">
+            <label for="fname">Förnamn</label>
+          </div>
+          <div class="col-75">
+            <input type="text" id="fname" name="fname" placeholder="Förnamn">
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-25">
+            <label for="lname">Efternamn</label>
+          </div>
+          <div class="col-75">
+            <input type="text" id="lname" name="lname" placeholder="Efternamn">
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-25">
+            <label for="question">Min fråga gäller...</label>
+          </div>
+          <div class="col-75">
+            <select id="question" name="question">
+              <option value="välj">Välj...</option>
+              <option value="medlem">Medlemsskap</option>
+              <option value="sponsor">Sponsor</option>
+              <option value="event">Event</option>
+              <option value="styrelse">Styrelse</option>
+              <option value="övrigt">Övrigt</option>
+            </select>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-25">
+            <label for="subject">Meddelande</label>
+          </div>
+          <div class="col-75">
+            <textarea id="subject" name="subject" placeholder="Skriv ditt meddelande här..." style="height:200px"></textarea>
```

Comment:
And the styling with height we can put in the CSS styling instead, which is the better practise 🚀